docs/content/stable/api/ysql/exprs/geo_partitioning_helper_functions/func_yb_server_zone.md

```diff
@@ -31,7 +31,7 @@ yugabyte=# SELECT yb_server_zone();
 
 ## Usage in Row-level geo-partitioning
 
-Similar to [`yb_server_region()`](../func_yb_server_region), this function is also helpful while implementing [Row-level geo-partitioning](../../../../../explore/multi-region-deployments/row-level-geo-partitioning/), as it can significantly simplify inserting rows from the user server's partition if the partitioning is based on default value of yb_server_zone().
+Similar to [`yb_server_zone()`](../func_yb_server_region), this function is also helpful while implementing [Row-level geo-partitioning](../../../../../explore/multi-region-deployments/row-level-geo-partitioning/), as it can significantly simplify inserting rows from the user server's partition if the partitioning is based on default value of yb_server_zone().
```

Comment:
```suggestion
Similar to [`yb_server_region()`](../func_yb_server_region), this function is also helpful while implementing [Row-level geo-partitioning](../../../../../explore/multi-region-deployments/row-level-geo-partitioning/), as it can significantly simplify inserting rows from the user server's partition if the partitioning is based on default value of yb_server_zone().
```